R/model.R

```diff
@@ -192,6 +192,11 @@ compile_method <- function(threads = FALSE,
     build_cleanup(exe,
                   remove_main = TRUE)
   }
+  # add path to the build tbb library to the PATH variable to avoid copying the dll file
+  if (os_is_windows()) {
+    path_to_TBB <- file.path(cmdstan_path(), "stan", "lib", "stan_math", "lib", "tbb")
+    Sys.setenv(PATH = paste0(path_to_TBB, ";", Sys.getenv("PATH"),sep=""))
```

Comment:
This looks right, but just checking, were you able to confirm on windows that this works?

Reply:
Yes, I double checked that the .dll file is not copied.

Reply:
This is where I wanted to put that version check in with the last commit. This doesnt need to be done for 2.20 and earlier. It doesnt really hurt, but still unnecessary. Fixed now.